### DISCUSSION TITLE
Airflow integration

### DISCUSSION THREAD

**Discussion by scheung38** [fastapi/fastapi#6884] [Questions] [answered]
Use I integrated FASTAPI instead of Flask for Airflow, but under the hood Airflow is using Flask, 
so not sure how to use FASTAPI instead?

**Comment by Kludex** [comment, marked as answer]
For now: https://fastapi.tiangolo.com/advanced/wsgi/#using-wsgimiddleware

Code example:
```python
from airflow.www.app import create_app
from fastapi import FastAPI
from fastapi.middleware.wsgi import WSGIMiddleware

app = FastAPI()
app.mount("/flask", WSGIMiddleware(create_app())
```

**Comment by scheung38** [comment]
Then it is not using ASGI then?

**Comment by tiangolo** [comment]
Sorry, I can't help you if you don't follow the issue template and provide more information.

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
